externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/operation/ExecuteStatement.scala

```diff
@@ -139,6 +146,58 @@ class ExecuteStatement(
     setState(OperationState.FINISHED)
   }
 
+  private def runSetOperation(setOperation: SetOperation): Unit = {
+    if (setOperation.getKey.isPresent) {
+      val key: String = setOperation.getKey.get.trim
+
+      if (setOperation.getValue.isPresent) {
+        val newValue: String = setOperation.getValue.get.trim
+        executor.setSessionProperty(sessionId, key, newValue)
+      }
+
+      val value = executor.getSessionConfigMap(sessionId).getOrDefault(key, "")
```

Comment:
So looks that we support `set`, `set k=v` and `set k`, and the last one is for retrieving a single value from a matched key? shall we also test it?

Reply:
Yes, but Flink doesn't support `set k` at the moment, so I skip the test for it.

Reply:
will `set k` passes the parsing phase or fail early before entering this func?

Reply:
Yes, it will cause a SQL parse error.

Reply:
then I am OK with this